### ISSUE TITLE
[AccessLint][a11y] homepage logo link missing text for assistive tech

### ISSUE THREAD

**Issue by ckundo** [rails/website#32]
The Rails logo link on the homepage has no text for assistive technology.

This is an issue for screen reader users and dictation users who are navigating the page.

Suggested solution is to use an `aria-label='Home'` on the link element.

```
<a class="nav__logo" href="/" aria-label="Home"></a>
```

[Issue found via AccessLint](https://airtable.com/shrqNrIt8NJvViAem/tblOfEFozRYQQ6qm1/viwtsmZwCTpzLpSRq/recCEXHah4LIMKWNY)

**Comment by ckundo** [comment]
happy to open a PR for this...

**Comment by seanmitchell** [comment]
@ckundo I just pushed a little fix for this.

**Comment by ckundo** [comment]
[fixed by ba5a9608]
